.github/workflows/k8s-integration-tests.yaml

```diff
@@ -21,6 +21,9 @@ jobs:
     steps:
       - name: Checkout code
         uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+          persist-credentials: false
```

Comment:
This job needs the full git history to correctly generate the `prefect` version. I saw flow run failures in the integration tests due to client/server version mismatch.